services/exec/client/client.go

```diff
@@ -58,21 +65,53 @@ func (p *execCmd) Execute(ctx context.Context, f *flag.FlagSet, args ...interfac
 	return c.Execute(ctx, args...)
 }
 
-type runCmd struct{}
+type runCmd struct {
+	streaming bool
+
+	// returnCode internally keeps track of the final status to return
+	returnCode subcommands.ExitStatus
+}
 
 func (*runCmd) Name() string     { return "run" }
 func (*runCmd) Synopsis() string { return "Run provided command and return a response." }
 func (*runCmd) Usage() string {
-	return `run <command> [<args>...]:
+	return `run <command> [--streaming] [<args>...]:
```

Comment:
bikeshed: "--stream" seems equivalently evocative, and is shorter to type (feel free to leave as-is if you disagree)

Reply:
I like shorter flag names, changed